plugin/core/windows.py

```diff
@@ -243,6 +243,9 @@ def start_async(self, config: ClientConfig, initiating_view: sublime.View) -> No
             # debug('Already starting on this window:', config.name)
             return
         try:
+            # Update folders in case those have been changed by the user. There is no currently
+            # no notification in ST that would notify about changed folder list.
+            self._workspace.update()
```

Comment:
Should be done earlier actually, before the checks that check if the file is part of the workspace.